ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2

```diff
@@ -309,6 +309,10 @@ host_monitoring_cron:
   minute: "*/5"
   job: ops-runner -f -s 60 -n ozc.send.synthetic.heartbeat.quick /usr/bin/ops-zagg-client -k heartbeat.ping -o 1 --synthetic
 
+- name: Delete stuck projects for bz 1367432
+  minute: "*/5"
```

Comment:
I am mostly interested in ensuring that the cleanup delay is long enough to allow any timeouts to occur and not get in the way of the regular project/resource creation. Once the user is stuck, waiting for 10 minutes or 20 minutes won't make that big of a difference.